src/main/java/jewellery/inventory/dto/request/TransferResourceRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package jewellery.inventory.dto.request;
+
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@Data
+@RequiredArgsConstructor
+public class TransferResourceRequestDto {
+
+    private static final String QUANTITY_MIN_VALUE_MSG = "Quantity should not be less than 1.";
```

Comment:
Quantity can be less than 1, e.g. 0.5. I see the min value is actually 0.

Reply:
Done!